src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeForceCaseProgressionToCaseUnderReviewPersonalisationTest.java

```diff
@@ -36,6 +36,7 @@ public class LegalRepresentativeForceCaseProgressionToCaseUnderReviewPersonalisa
     private String legalRepRefNumber = "somelegalRepRefNumber";
     private String appellantGivenNames = "someAppellantGivenNames";
     private String appellantFamilyName = "someAppellantFamilyName";
+    private String linkToOnlineService = "https://immigration-appeal.demo.platform.hmcts.net/start-appeal";
```

Comment:
check environment pointing only to demo?

Reply:
from the look of it, its just to give a value to the mock, it could in theory equal anything